WoLua/Lua/Api/Game/PlayerApi.cs

```diff
@@ -252,7 +252,6 @@ internal PlayerApi(ScriptContainer source) : base(source) { }
 	[LuaDoc("Provides three values consisting of the current character's X (east/west), Y (north/south), and Z (vertical) coordinates.",
 		"If the player isn't loaded, all three values will be nil.",
 		"This property is shorthand for `.Entity.MapCoords`.")]
-	[return: AsLuaType(LuaType.Number)]
```

Comment:
Why was this attribute removed?

Reply:
Didn't do anything I think
> CS0657: 'return' is not a valid attribute location for this declaration. Valid attribute locations for the declaration are 'properties'. All attributes in this block will be ignored.

Reply:
Ah, damn. Gonna have to figure out how to fix that, since the documentation needs a way to specify the lua return type...